Documentation/ContentElements/Content/Index.rst

```diff
@@ -1,74 +1,198 @@
+
+This commonly happens when you switched the
+`Backend layout in the page properties <https://docs.typo3.org/permalink/t3editors:pages-properties-appearance>`_.
+
+It can also happen after imports or updates. "Unused content" is usually not
+displayed on the page. You can restore it if it is still needed or delete it if
+it is not needed anymore.
+
+The "Unused content" is found at the bottom of the page module. you can move it
+via drag an drop to the desired location or delete it, if it is not needed anymore.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/MoveUnusedContent.png
+    :alt: Demonstration of moving unused content back into a normal content area
+
+..  _content-working-troubleshooting-recycler:
+
+Restore deleted content elements using the Recycler
+---------------------------------------------------
+
+If you accidentally delete a content element or even a complete page you can
+restore your data using the module :guilabel:`Web > Recycler` if you have
+sufficient permissions and it is installed.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/RestoreContent.png
+    :alt: Screenshot of the Recycler module in the TYPO3 backend, demonstrating how to restore a deleted content element
+
+See also :ref:`Recycler manual, For editors <typo3/cms-recycler:for-editors>`.
+
+..  _content-working-troubleshooting-history:
+
+Undo changes using the Page History
+-----------------------------------
+
+You can use the page history to undo any kind of content changes. If you have
+no access to the recycler you can also use it to restore deleted content.
+
+Use the content menu on the page tree and choose "History/Undo".
+
+..  figure:: /Images/ManualScreenshots/ContentElements/RestoreContent.png
+    :alt: Screenshot of the page history in the TYPO3 backend
+
+    Use the "Undo" button to undo your changes
+
+In this module you can even see a detailed view on how text or other fields
+where changed and who made which changes when.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/ContentElementHistoryDifferences.png
+    :alt: Screenshot of the page history in the TYPO3 backend
+
+    Compare the differences. Use the undo button to undo changes.
+
+..  _content-working-began-editing:
 
-.. figure:: ../../Images/ManualScreenshots/Page/PageModule/PageModuleMoveElements.png
-   :alt: Moving content elements around using drag and drop
-   :class: with-border
+The BE-User 'xxx' began to edit this record x min ago
+-----------------------------------------------------
 
-Click in the grey bar at the top of a content element to start dragging it.
-Highlighted areas will then show you where you can drop it.
+It is not possible in TYPO3 for two editors to work on the same content element
+at the same time. The last editor who saves the content element will override
+all other changes. You should therefore avoid working on content elements that
+another editor is currently working on. These are marked with a warning:
```

Comment:
Not a must have: This message will disappear after 2 hours of other users inactivity.
No need to overcomplicate things ;-)

Reply:
I think that will over complicate things.